### ISSUE TITLE
Integral refactoring

### ISSUE THREAD

**Issue by fedepedron** [MALBECC/lio#237]
I open this issue for the sake of bookkeeping the process started with #216 and #236. These are, I believe, the steps to be taken:

[ ] Refactor int1 and int1G
[ ] Refactor int2 and int3lu
[ ] Refactor int2G and int3G (THE HORROR)
[ ] Refactor intsol and intsolG
[ ] Refactor intSG and intfld
[ ] Either move all basis set data (nuc, a, c, d, ncont, nshell, norm) to the faint_module or create a basis_data module to keep these separately.

**Comment by ramirezfranciscof** [comment]
Doesn't the basis_data module already exist? Probably doesn't have all the information though...

**Comment by fedepedron** [comment]
Yeah, I believe it was one of those modules that were like, deprecated from the craddle. Like liosubs I think.

**Comment by ramirezfranciscof** [comment]
> I believe it was one of those modules that were like, deprecated from the craddle

How dare you!

Nah, I'm pretty sure that I use it to calculate the integrals for the forces of ehrenfest.

**Comment by fedepedron** [comment]
> I'm going to mess all over your tick boxes...
You monster!
